code/modules/client/client_colour.dm

```diff
@@ -9,9 +9,9 @@
  * so that the "highest priority" one is always displayed as opposed to the default of
  * "whichever was set last is displayed".
  *
- * Refactored to allow multiple overlapping client colours
```

Comment:
i'd prefer these left in, changing UK to US spelling is a can of worms. same goes for behaviour, I hate it but US/UK is too fine a hair to split